x-pack/plugins/ml/public/application/jobs/components/jobs_action_menu.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiButtonEmpty } from '@elastic/eui';
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useMlLocator, useNavigateToPath } from '../../contexts/kibana';
+import { ML_PAGES } from '../../../../common/constants/locator';
+
+export const JobsActionMenu: FC = () => {
```

Comment:
Could this be put in a folder called `jobs_action_menu` please?
With an `index.ts` to export it.
So it is constant with the general pattern we've been using for other components.

Reply:
Done in https://github.com/elastic/kibana/pull/121652/commits/888047eb32c27808adc87dd46089756ce8cd156b